src/output.jl

```diff
@@ -304,11 +290,6 @@ end
             out
         end
 
-        # default text/plain passes to html... handles Interact issues
```

Comment:
Unclear what issue this fixed and if it is still needed, introduced in 16a2e34e950a020746cddf9ed964275a9f131ea3